requirements.txt

```diff
@@ -1,3 +1,2 @@
-asgiref==3.7.2
-Django==4.2.6
-sqlparse==0.4.4
+# Supported 3.2, 4.0, 4.1, 4.2, 5.0, 5.1
+Django>=3.2
```

Comment:
Django 3.2 LTS has ended. It should be 4.2+.

Reply:
Indeed. Made small fix https://github.com/paqstd-dev/django-suspense/pull/3